src/components/landing-page/create-production.tsx

```diff
@@ -97,6 +122,32 @@ export const CreateProduction = () => {
     }
   }, [createdProductionId, dispatch, reset]);
 
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const handleCopyProdUrlsToClipboard = (input: any) => {
+    if (input !== null) {
+      navigator.clipboard
+        .writeText(input)
+        .then(() => {
+          let timeout: number | null = null;
+          setCopiedUrl(true);
+          console.log("Text copied to clipboard");
+
+          timeout = window.setTimeout(() => {
+            setCopiedUrl(false);
+          }, 1500);
+
+          return () => {
+            if (timeout !== null) {
+              window.clearTimeout(timeout);
+            }
+          };
```

Comment:
You're not inside of a `useEffect` here, so this doesn't do what I think you think it does :sweat_smile: 

What is happening here is that you're returning a function, that will never be executed, from an unhandled promise